mitiq/zne/tests/test_zne.py

```diff
@@ -630,3 +630,50 @@ def executor(circuit):
         scale_noise=noise_scaling_method,
     )
     assert np.isclose(zne_res, two_stage_zne_res)
+
+
+@pytest.mark.parametrize(
+    "extrapolation_factory", [RichardsonFactory, LinearFactory]
+)
+@pytest.mark.parametrize(
+    "to_frontend",
+    [None, to_qiskit, to_braket, to_pennylane, to_pyquil, to_qibo],
+)
+def test_two_stage_zne_default_scaling(extrapolation_factory, to_frontend):
```

Comment:
`to_frontend` makes it seem like there should be an argument `frontend` that is going to be passed.

```suggestion
def test_two_stage_zne_default_scaling(extrapolation_factory, conversion_func):
```